### PR TITLE
get site by id

### PR DESCRIPTION
**Pull request by joshplumley**

### CODE REVIEW

File: src/site/controller.ts [hunk 1]
```diff
@@ -6,10 +6,15 @@
 import { Request, Response, Router } from 'express';
 import { ExpressHandler, HTTPError } from '../utils';
 import { SiteModel } from './model';
+import { verifySiteId } from './utils';
 
 const SiteRouter = Router();
 export default SiteRouter;
 
+SiteRouter.get(['/:siteId'], (res, req, next) =>
+  verifySiteId(res, req, next, 'param'),
```
Comment (jbolles94) [inline src/site/controller.ts:15]: This can be replaced by the more generic verison once my MR is in.

File: tests/unit/site.spec.ts [hunk 2]
```diff
@@ -38,23 +38,43 @@ describe('# SITE', () => {
...
-        name: 'TEST SITE',
-        location: 'TEST',
-        timezone: 'EST',
-      });
+  });
 
-      const site = await TEST_DB_CLIENT.db()
-        .collection(SiteModel.collection.name)
-        .findOne({ name: 'TEST SITE' });
+  it('Should get a single id with :siteId.', async () => {
```
Comment (jbolles94) [inline tests/unit/site.spec.ts:43]: Do we need a test for the failures of getting a site?

File: src/site/controller.ts (hunk 1 above)
Comment (joshplumley) [reply src/site/controller.ts]: done

File: tests/unit/site.spec.ts (hunk 2 above)
Comment (joshplumley) [reply tests/unit/site.spec.ts:58]: yes

File: src/site/controller.ts [hunk 3]
```diff
@@ -11,6 +11,10 @@ import { SiteModel } from './model';
 const SiteRouter = Router();
 export default SiteRouter;
 
+SiteRouter.get(['/:siteId'], (req, res, next) =>
```
Comment (jbolles94) [inline src/site/controller.ts:14]: Async/awaits

Comment (joshplumley) [reply src/site/controller.ts]: Done

**Review by jbolles94 — APPROVED** [review]

**Review by jbolles94 — APPROVED** [review]

**Pull request merged**